applications/FluidDynamicsApplication/tests/levelset_consistent_nodal_gradient_test.py

```diff
@@ -0,0 +1,77 @@
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import KratosMultiphysics
+import KratosMultiphysics.FluidDynamicsApplication as KratosCFD
+import math
+import os
+
+# from KratosMultiphysics.gid_output_process import GiDOutputProcess
+
+def GetFilePath(fileName):
+    return os.path.join(os.path.dirname(os.path.realpath(__file__)), fileName)
+
+class TestConsistentLevelsetNodalGradient(KratosUnittest.TestCase):
```

Comment:
I think you didn't add this test to the `FluidDynamicsApplication` suites.

Reply:
Added.